data/models/TimesheetRecord.js

```diff
@@ -65,6 +65,13 @@ module.exports = (sequelize, DataTypes) => {
             as: 'project',
             foreignKey: 'projectId'
         });
+
+        TimesheetRecord.belongsToMany(models.Company, {
```

Comment:
Wrong type of association
Company has many timesheet records
Timesheet record belongs to a company

Reply:
fix